app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please make sure the devise is `confirmable` and ask for the confirmation of the email as requested in the project requirements.

![image](https://github.com/francksefu/blog_web/assets/83803414/2e2d0c76-8d95-46c2-9f74-b00f6b8acba8)